### PR TITLE
Restructuring, new commands and resolving some bugs

### PR DESCRIPTION
**Pull request by vcidst**
This commit only changes the `%` based string formatting to use of `.format()` function. I am unable to test this locally so please make sure this compiles before merging to master.

### CODE REVIEW

**Comment by mmcmd** [comment]
Absolutely amazing, I'll check it out tomorrow night

File: verif.py [hunk 1]
```diff
@@ -58,36 +58,53 @@
...
                     await message.add_reaction('✅')
-                    logging.info(message.author.name + " (%d) has been verified. Account age: %d days" % (author_id,account_age_days))
+                    logging.info("{0} ({1}) has been verified. Account age: {2} days".format(message.author.name, author_id,account_age_days))
                     return
             else:
-                await message.channel.send("You're already able to speak in the server, there is no need for you to get verified. Contact <@%s> (Moderator Mail, top of the member list) if there is an issue."% str(moderator_mail))
-                logging.info(message.author.name + " (%d) attempted to get verified, but doesn't have @ everyone as his top role. Verification aborted."% author_id)
+                await message.channel.send("You're already able to speak in the server, there is no need for you to get verified. Contact <@{0}> (Moderator Mail, top of the member list) if there is an issue.".format(moderator_mail))
+                logging.info("{} ({%d}) attempted to get verified, but doesn't have @ everyone as his top role. Verification aborted.".format(message.author.name,author_id))
```
Comment (vcidst) [inline verif.py]: This line is wrong and should be changed. How do I change this?

**Comment by vcidst** [comment]
I've added a few more changes do the following,

1. all message texts of the bot are taken from messages.json
2. I've created a resources folder that keeps this json file.
3. Changed the birthday command to birthdayboy
4. Added two more commands, birthdaygirl and birthdaynerd
5. The duration of birthday is also taken from messages.json
6. Currently it is set to 12 hours.

Issues addressed,
#21 , #18 , #19

**Review by vcidst — COMMENTED** [review]
This has been fixed in later commits

**Pull request merged**